src/components/dialog-full-screen/dialog-full-screen.pw.tsx

```diff
@@ -0,0 +1,565 @@
+      "rgb(255, 188, 25) 0px 0px 0px 3px, rgba(0, 0, 0, 0.9) 0px 0px 0px 6px"
+    );
+    const hasFocus = await focusedButton.focus();
+    expect(hasFocus).toBeFalsy();
+  });
+
+  test("should render DialogFullScreen component with help", async ({
+    mount,
+    page,
+  }) => {
+    await mount(
+      <DialogFullScreenComponent
+        title="Sample DialogFullScreen"
+        help="Some help text"
+      />
+    );
+
+    const helpIcon = getDataElementByValue(page, "question");
+    await helpIcon.hover();
+    await expect(tooltipPreview(page)).toHaveText("Some help text");
+  });
+
+  test("should render DialogFullScreen component with role", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DialogFullScreenComponent role="dialog" />);
+
+    await expect(
+      getDataElementByValue(page, "dialog-full-screen")
+    ).toHaveAttribute("role", "dialog");
+  });
+
+  test("should not render close icon when ShowCloseIcon is set to false", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DialogFullScreenComponent showCloseIcon={false} />);
+
+    await expect(getDataElementByValue(page, "close")).toHaveCount(0);
+  });
+
+  test("should render close icon when ShowCloseIcon is set to true. When you click the CloseIcon the Dialog is closed", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DialogFullScreenComponent showCloseIcon />);
+
+    const fullDialog = getDataElementByValue(page, "dialog-full-screen");
+    await expect(fullDialog).toHaveCount(1);
+    const closeIcon = getDataElementByValue(page, "close").nth(1);
+    await closeIcon.click();
+    await expect(fullDialog).toHaveCount(0);
+  });
+
+  test("should render Dialog Full Screen with header children", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DialogFullScreenWithHeaderChildren />);
+
+    await expect(getComponent(page, "pill").nth(0)).toHaveText("A pill");
+    await expect(getComponent(page, "pill").nth(1)).toHaveText("Another pill");
+  });
+
+  test("should render Dialog component with content padding disabled", async ({
+    mount,
+    page,
+  }) => {
+    await mount(
+      <DialogFullScreenComponent
+        title="Dialog Full Screen Title"
+        disableContentPadding
+      />
+    );
+
+    const content = getDataElementByValue(page, "content");
+    await expect(content).toHaveCSS("padding-right", "0px");
+    await expect(content).toHaveCSS("padding-bottom", "0px");
+  });
+
+  // skip this test for now as FE-6053 has not been fixed yet
+  test.skip("should render DialogFullScreen component with DisableClose prop", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DialogFullScreenComponent disableClose />);
+
+    const closeIcon = getDataElementByValue(page, "close").nth(1);
+    await closeIcon.click();
+    await expect(getDataElementByValue(page, "dialog-full-screen")).toHaveCount(
+      1
+    );
+  });
+});
+
+test.describe("should check accessibility for Dialog Full Screen", () => {
```

Comment:
**nitpick/suggestion (non-blocking):** It just means we wouldn't have two lots of `should check accessibility`

```suggestion
test.describe("Accessibility for Dialog Full Screen", () => {
```